File: branches/simosws/simosws/BusinessLayer/DTO/Respuesta.vb
Public Class Respuesta

    Private _oprespuestaId As Integer
    Public Property opcionRespuestaId() As Integer
        Get
            Return _oprespuestaId
        End Get
        Set(ByVal value As Integer)
            _oprespuestaId = value
        End Set
    End Property

    Private _encuestadoId As Integer
    Public Property encuestadoId() As Integer
        Get
            Return _encuestadoId
        End Get
        Set(ByVal value As Integer)
            _encuestadoId = value
        End Set
    End Property

    Private _respuestaId As Integer
    Public Property respuestaId() As Integer
        Get
            Return _respuestaId
        End Get
        Set(ByVal value As Integer)
            _respuestaId = value
        End Set
    End Property

    Private _respuestaPadreId As Integer
    Public Property respuestaParentId() As Integer
        Get
            Return _respuestaPadreId
        End Get
        Set(ByVal value As Integer)
            _respuestaPadreId = value
        End Set
    End Property

    Private _respuestaTexto As String
    Public Property respuestaTexto() As String
        Get
            Return _respuestaTexto
        End Get
        Set(ByVal value As String)
            _respuestaTexto = value
        End Set
    End Property

    Private _respuestaNum As Double
    Public Property respuestaNumero() As Double
        Get
            Return _respuestaNum
        End Get
        Set(ByVal value As Double)
            _respuestaNum = value
        End Set
    End Property


    Private _preguntaId As String
    Public Property preguntaId() As String
        Get
            Return _preguntaId
        End Get
        Set(ByVal value As String)
            _preguntaId = value
        End Set
    End Property


    Private _medins As String
    Public Property prescripcionId() As String
        Get
            Return _medins
        End Get
        Set(ByVal value As String)
            _medins = value
        End Set
    End Property


    Private _child As List(Of Respuesta)
    Public Property child() As List(Of Respuesta)
        Get
            Return _child
        End Get
        Set(ByVal value As List(Of Respuesta))
            _child = value
        End Set
    End Property

End Class

File: branches/simos/src/gob/sis/simos/entity/Respuesta.java
package gob.sis.simos.entity;

import java.io.Serializable;
import java.util.List;

import com.google.gson.annotations.Expose;
import com.j256.ormlite.field.DatabaseField;
import com.j256.ormlite.table.DatabaseTable;

@DatabaseTable(tableName="encta_rsptas")
public class Respuesta implements Serializable {

	private static final long serialVersionUID = 1L;
	//ID
	@DatabaseField(columnName="id")
	@Expose
	private Integer respuestaId;
	
	@DatabaseField(columnName="preg_id")
	@Expose
	private Integer preguntaId;
	
	@DatabaseField(columnName="op_rsp_id")
	@Expose
	private Integer opcionRespuestaId;
	
	@DatabaseField(columnName="encstdo_id")
	private Integer encuestadoId;
	
	@DatabaseField(columnName="preg_prnt_id")
	private Integer respuestaParentId;
	
	@DatabaseField(columnName="rsp_text")
	@Expose
	private String respuestaTexto;
	
	@DatabaseField(columnName="rsp_number")
	@Expose
	private Double respuestaNumero;
	
	@DatabaseField(columnName="rsp_prd_id")
	@Expose
	private String prescripcionId;
	
	@Expose
	private Integer preguntaParentId;
	
	@Expose
	private List<Respuesta> child;
	
	
	public Integer getPreguntaParentId() {
		return preguntaParentId;
	}
	public void setPreguntaParentId(Integer preguntaParentId) {
		this.preguntaParentId = preguntaParentId;
	}
	public List<Respuesta> getChild() {
		return child;
	}
	public void setChild(List<Respuesta> child) {
		this.child = child;
	}
	public Integer getOpcionRespuestaId() {
		return opcionRespuestaId;
	}
	public void setOpcionRespuestaId(Integer opcionRespuestaId) {
		this.opcionRespuestaId = opcionRespuestaId;
	}
	public Integer getEncuestadoId() {
		return encuestadoId;
	}
	public void setEncuestadoId(Integer encuestadoId) {
		this.encuestadoId = encuestadoId;
	}
	public Integer getRespuestaId() {
		return respuestaId;
	}
	public void setRespuestaId(Integer respuestaId) {
		this.respuestaId = respuestaId;
	}
	public Integer getRespuestaParentId() {
		return respuestaParentId;
	}
	public void setRespuestaParentId(Integer respuestaParentId) {
		this.respuestaParentId = respuestaParentId;
	}
	public String getRespuestaTexto() {
		return respuestaTexto;
	}
	public void setRespuestaTexto(String respuestaTexto) {
		this.respuestaTexto = respuestaTexto;
	}
	public Double getRespuestaNumero() {
		return respuestaNumero;
	}
	public void setRespuestaNumero(Double respuestaNumero) {
		this.respuestaNumero = respuestaNumero;
	}
	public String getPrescripcionId() {
		return prescripcionId;
	}
	public void setPrescripcionId(String prescripcionId) {
		this.prescripcionId = prescripcionId;
	}
	public Integer getPreguntaId() {
		return preguntaId;
	}
	public void setPreguntaId(Integer preguntaId) {
		this.preguntaId = preguntaId;
	}
	
	
}
